src/App.css

```diff
@@ -28,11 +28,33 @@
   color: #61dafb;
 }
 
-@keyframes App-logo-spin {
+.App-main {
+  background-color: yellow;
+  min-height: 90vh;
+  display: inline-flexbox;
+  flex-direction: column;
+  justify-content:center
+}
+
+.Json-input-container {
```

Comment:
avoid using classes for components for now.
use styles inline

Reply:
👍